src/components/markdown-renderer/markdown-extension/gist/replace-legacy-gist-short-code.ts

```diff
@@ -7,7 +7,7 @@
 import type { RegexOptions } from '../../../../external-types/markdown-it-regex/interface'
 import { GistMarkdownExtension } from './gist-markdown-extension'
 
-const finalRegex = /^{%gist (\w+\/\w+) ?%}$/
+const finalRegex = /^{%gist\s+(\w+\/\w+)\s+%}$/
```

Comment:
HedgeDoc 1 uses `\s*` before the "closing tag". So the space was not needed explicitly.
The usage of `\s+` requires at least one explicit space before the closing tag.
This might be breaking existing documents. Sure, we want to do that?